src/frontends/pytorch/src/op/expand.cpp

```diff
@@ -42,24 +42,30 @@ OutputVector translate_expand_as(const NodeContext& context) {
 };
 
 OutputVector translate_expand_fx(const NodeContext& context) {
-    // aten::expand(Tensor(a) self, SymInt[] size, *, bool implicit=False) -> Tensor(a)
-    num_inputs_check(context, 2, 3);
+    auto num_inputs = context.get_input_size();
+    num_inputs_check(context, 2, num_inputs);
```

Comment:
Is this assertion still valid if we're getting `num_inputs` from the `context` itself?

Reply:
Checking the minimum number of inputs (which is at least 2). But probably it's very unlikely to be less then 2.